test-data/recipes/interpreter-detection/nu/recipe.yaml

```diff
@@ -0,0 +1,16 @@
+package:
+  name: test-interpreter-nu
+  version: 0.1.0
+
+build:
+  script: build.nu
+
+requirements:
+  build:
+    - nushell
+
+tests:
+  - script: build.nu
+    requirements:
+      run:
```

Comment:
Wrt to removing `nushell` from pixi, would it work if you use `build` here, instead of run?